Documentation build fails with `UTF-8 and US-ASCII`

```
[INFO] --- asciidoctor:2.2.2:process-asciidoc (html) @ spring-data-couchbase ---
2023-02-17T10:32:41.424Z [main] WARN FilenoUtil : Native subprocess control requires open access to the JDK IO subsystem
Pass '--add-opens java.base/sun.nio.ch=ALL-UNNAMED --add-opens java.base/java.io=ALL-UNNAMED' to enable.
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Using 'UTF-8' encoding to copy filtered properties files.
[INFO] Copying 0 resource
[INFO] asciidoctor: WARN: template.adoc: line 92: id assigned to section already in use: template.sub-document-ops
[INFO] asciidoctor: INFO: possible invalid reference: projections
[INFO] asciidoctor: INFO: possible invalid reference: projections
[INFO] asciidoctor: INFO: possible invalid reference: couchbase.repository.multibucket
[INFO] asciidoctor: INFO: possible invalid reference: couchbase.repository.multibucket
Feb 17, 2023 10:32:52 AM org.asciidoctor.jruby.internal.JRubyAsciidoctor convertFile
SEVERE: (CompatibilityError) incompatible character encodings: UTF-8 and US-ASCII
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  31.589 s
[INFO] Finished at: 2023-02-17T10:32:52Z
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.asciidoctor:asciidoctor-maven-plugin:2.2.2:process-asciidoc (html) on project spring-data-couchbase: Execution html of goal org.asciidoctor:asciidoctor-maven-plugin:2.2.2:process-asciidoc failed: org.jruby.exceptions.EncodingError$CompatibilityError: (CompatibilityError) incompatible character encodings: UTF-8 and US-ASCII -> [Help 1]

```

--- a/src/main/asciidoc/index.adoc
+++ b/src/main/asciidoc/index.adoc
@@ -1,5 +1,5 @@
 = Spring Data Couchbase - Reference Documentation
-Michael Nitschinger, Oliver Gierke, Simon Baslé, Michael Reiche
+Michael Nitschinger, Oliver Gierke, Simon Basle, Michael Reiche
 :revnumber: {version}
 :revdate: {localdate}
 :spring-data-commons-docs: ../../../../spring-data-commons/src/main/asciidoc

